spec/models/exception_hunter/error_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+module ExceptionHunter
+  describe Error do
+    subject { build(:error) }
+
+    describe 'validations' do
+      it { is_expected.to validate_presence_of(:class_name) }
+    end
+
+    describe 'ocurred at' do
+      let!(:error) { create(:error, ocurred_at: nil) }
+
+      context 'when creating' do
+        it 'sets timestamp' do
+          expect(error.ocurred_at).to be_within(1.second).of(Time.now)
+        end
+      end
+
+      context 'when updating' do
+        it "doesn't modify timestamp" do
+          expect { error.update!(class_name: 'AnotherOne') }
+            .not_to change { error.reload.ocurred_at }
```

Comment:
🤘

Just FYI, you can also do `not_to change(error, :ocurred_at)` which looks cooler